### DISCUSSION TITLE
Information missing or not interpreted on my Watch

### DISCUSSION THREAD

**Discussion by Dnstouron** [NightscoutFoundation/xDrip#2377] [Q&A]
Hello
I have been discovering xdrip+ for a few weeks and the application is great.
Having your blood sugar on my mobile display is great!! Thank you very much for your work
I recently acquired a blackview watch (R7 pro) which is not WearOS.
It requires H band software (not 2.0) to send or receive information from my phone (BVA 6300)
the problem is that the information does not come back correctly from xdrip+ I have this on my watch
10/12 15:00
xx:Delta: -1mg/dl
Do I have to look at Xdrip+ to make changes? or is it rather the H band application that is the problem?
Xdrp+version
9c29f3a-2022.10.05
Data source limiter
H-Band version 8.4.5

Thanks for your help

**Comment by Navid200** [comment]
Have you asked watchdrip?

**Reply by Dnstouron** [reply]
You made me discover Watchdrip, I didn't know.
From what I've read it's for watches with the Mifit app. Which is not the case with mine who uses this application: https://play.google.com/store/apps/details?id=com.veepoo.hband&hl=fr&gl=US

**Reply by Navid200** [reply]
Talk to him.
He is the most active developer supporting watches.  
If the watch you have is popular, he may add support.

**Reply by Dnstouron** [reply]
My watch it's not a MiBand watch.. it's an other "WatchOs", I test Watchdrip but the watch is not reconize

**Reply by Navid200** [reply]
I suggest that you ask the individual who supports watchdrip.
That's your best chance of getting support for your watch with xDrip.

**Comment by Dnstouron** [comment]
thanks .. i'll try other watch application on my smartphone because xdrip+ is good

**Reply by Dnstouron** [reply]
I'll try a lot of app on playstore .. no way to connect my watch :(
I tested the alerts under xdrip+ and all the characters of the alerts are displayed correctly on my watch, as well as the blood sugar level which is absent on the other shipments.
There must be a character/parameter that is not interpreted in the sends made every 5 minutes on my watch by xdrip
Maybe I need to go see in the code what's wrong with my watch
